src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="page">
+      <div class="calendar calendar--start-day_sun calendar--month-length_31">
```

Comment:
Follow BEM naming for modifier **block-name--modifier-name--modifier-value**
```suggestion
      <div class="calendar calendar--start-day--sun calendar--month-length--31">
```